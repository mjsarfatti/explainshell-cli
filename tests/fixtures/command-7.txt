tar xzvf archive.tar.gz