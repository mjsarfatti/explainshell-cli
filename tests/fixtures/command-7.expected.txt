tar

    The GNU version of the tar archiving utility

    [ tar -> https://explainshell.com/explain/1/tar ]

x

    -x, --extract, --get
          extract files from an archive

z

    -z, --gzip, --gunzip --ungzip

v

    -v, --verbose
          verbosely list files processed

f archive.tar.gz

    -f, --file ARCHIVE
          use archive file or device ARCHIVE 